src/index.ts

```diff
@@ -109,11 +109,14 @@ class DebuggerHandler<H extends ConsoleHandler | NotebookHandler> {
     });
 
     debug.model.disposed.connect(async () => {
-      await debug.stop();
-      Object.keys(this.handlers).forEach(id => {
-        this.handlers[id].dispose();
-      });
-      this.handlers = {};
+      const idsOfHandler = Object.keys(this.handlers);
```

Comment:
How about `handlerIds` instead of `idsOfHandler`?

Reply:
Resolved